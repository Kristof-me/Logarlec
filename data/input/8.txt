load map.data
create student s1 r1
create student s2 r1
create janitor j1 r2
create gas-mask g1 s2
effect gas r1
move j1 d12
status s1 location
status s2 location
status r1 effects