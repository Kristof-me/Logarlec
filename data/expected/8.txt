r1
r2
ragad: 10
